DuggaSys/sectioned.php

```diff
@@ -681,7 +681,84 @@ function updateGithubDir($pdo, $githubDir, $lid) {
 				
 		</div>
 
-
+<?php
+//Insert into gitRepo DB
+function insertIntoSqLiteGitRepo($cid, $githubURL){
+	$pdolite = new PDO('sqlite:../../githubMetadata/metadata2.db');
+	$query = $pdolite->prepare("INSERT OR REPLACE INTO gitRepos (cid, repoURL) VALUES (:cid, :repoURL)"); 
+	$query->bindParam(':cid', $cid);
+	$query->bindParam(':repoURL', $githubURL);
+	if (!$query->execute()) {
+		$error = $query->errorInfo();
+		echo "Error updating file entries" . $error[2];
+		$errorvar = $error[2];
+		print_r($error);
+		echo $errorvar;
+    }else{
+		return true;
+	}
+}
+
+//Insert files into gitFiles DB
+function insertIntoSqLiteGitFiles($cid, $fileNames, $filePaths, $fileURLS, $downloadURL, $fileTypes, $SHA){
+	$pdoLite = new PDO('sqlite:../../githubMetadata/metadata2.db');
+	for($i = 0; $i < count($fileNames); $i++){
+		$query = $pdoLite->prepare('INSERT INTO gitFiles (cid, fileName, fileType, fileURL, downloadURL, fileSHA, filePath) VALUES (:cid, :fileName, :fileType, :fileURL, :downloadURL, :fileSHA, :filePath)');
```

Comment:
According to the [php-documentation](https://www.php.net/manual/en/control-structures.for.php) expression 2 of a for loop is evaluated at the beginning of each iteration. Consider moving `count($codeExamplesContent)` outside the for loop if possible.